app/models/measure_condition.rb

```diff
@@ -172,7 +174,19 @@ def unit_for(measure)
     end
   end
 
-private
+  def is_excluded_condition?
+    return true if universal_waiver_applies?
+    return false if negative_action? && is_threshold?
+    return false if negative_action? && action_code.in?(INCLUDED_NEGATIVE_ACTIONS)
+
+    negative_action? && document_code.blank?
+  end
```

Comment:
Thinking about how to simplify this.
Maybe this way? 🤔  (it just checks `negative_action?` once)
```
  def is_excluded_condition?
    return true if universal_waiver_applies?

    if negative_action?
      is_threshold? || action_code.in?(INCLUDED_NEGATIVE_ACTIONS)
    else
      document_code.blank?
    end
  end

```

Reply:
So we're covering these scenarios:

*globally excluded conditions*

1. universal waiver

*negative conditions that are included*

1. negative action and threshold is positive (really common)
2. negative action and the action code is 08 (common)

*negative conditions that are excluded*

1. negative action and document is blank (really common)